azavea/pwd-fishway
# Implement Swipe to continue

## Overview
Most of the application has all of its content on a screen, but some screens require the user to swipe to see additional content. We are adding a "Swipe to continue" helper text to make this more clear.

Connects #35 

### Demo
![Screen Shot 2019-09-06 at 2 25 10 PM](https://user-images.githubusercontent.com/5672295/64451489-562c2300-d0b2-11e9-94a6-888ce3e06215.png)
![Screen Shot 2019-09-06 at 2 25 36 PM](https://user-images.githubusercontent.com/5672295/64451490-562c2300-d0b2-11e9-88e8-39e2ed254d71.png)
![swipe_animation](https://user-images.githubusercontent.com/5672295/64451507-5debc780-d0b2-11e9-9a6e-3d2c909d8fe2.gif)

### Notes
- Swipe to continue disappears after a person sees the full animation on a tab, leaves, then returns to the same tab.  


## Testing Instructions
* `git pull`
* Enter app
* Note that animation plays and then disappears on the About tab
* Go to Meet the Fish
* Note that animation plays and then disappears
